pg_lakehouse/README.md

```diff
@@ -0,0 +1,219 @@
+- [x] Local file system
+- [ ] Azure Blob Storage (coming soon)
+- [ ] Google Cloud Storage (coming soon)
+- [ ] HDFS (coming soon)
+
+### Supported File Formats
+
+- [x] Parquet
+- [x] CSV
+- [x] JSON
+- [x] Avro
+- [ ] ORC (coming soon)
+
+### Supported Table Formats
+
+- [x] Deltalake
+- [ ] Apache Iceberg (coming soon)
+
+## Getting Started
+
+The following example uses `pg_lakehouse` to query an example dataset of NYC taxi trips from January 2024, hosted in a public S3 bucket provided by ParadeDB.
+
+```sql
+CREATE FOREIGN DATA WRAPPER s3_wrapper HANDLER s3_fdw_handler VALIDATOR s3_fdw_validator;
+
+-- Provide S3 credentials
+CREATE SERVER s3_server FOREIGN DATA WRAPPER s3_wrapper
+OPTIONS (region 'us-east-1', url 's3://paradedb-benchmarks', skip_signature 'true');
+
+-- Create foreign table
+CREATE FOREIGN TABLE trips (
+    trip_id             INT,
+    pickup_datetime     TIMESTAMP,
+    dropoff_datetime    TIMESTAMP,
+    pickup_longitude    DOUBLE PRECISION,
+    pickup_latitude     DOUBLE PRECISION,
+    dropoff_longitude   DOUBLE PRECISION,
+    dropoff_latitude    DOUBLE PRECISION,
+    passenger_count     SMALLINT,
+    trip_distance       REAL,
+    fare_amount         REAL,
+    extra               REAL,
+    tip_amount          REAL,
+    tolls_amount        REAL,
+    total_amount        REAL,
+    payment_type        INT,
+    pickup_ntaname      REAL,
+    dropoff_ntaname     REAL
+)
+SERVER s3_server
+OPTIONS (path 's3://paradedb-benchmarks/yellow_tripdata_2024-01.parquet', extension 'parquet');
+
+-- Success! Now you can query the remote Parquet file like a regular Postgres table
+SELECT COUNT(*) FROM trips;
+  count
+---------
+ 2964624
+(1 row)
+```
+
+## Query Acceleration
+
+On its own, `pg_lakehouse` is only able to push down column projections, sorts, and limits to DataFusion. This means that queries containing
+aggregates, joins, etc. are not fully accelerated.
+
+This can be solved by installing `pg_analytics`, which is able to push down the entirety of most queries over `pg_lakehouse` foreign tables to DataFusion.
+
+```sql
+\timing
+
+-- Query speed without pg_analytics
+SELECT COUNT(*) FROM hits;
+
+CREATE EXTENSION pg_analytics;
+
+-- Query speed with pg_analytics
+```
+
+## Object Stores
+
+### Local File System
+
+To be queryable, files must exist on the same machine as your Postgres instance.
+
+```sql
+CREATE FOREIGN DATA WRAPPER local_file_wrapper HANDLER local_file_fdw_handler VALIDATOR local_file_fdw_validator;
+CREATE SERVER local_file_server FOREIGN DATA WRAPPER local_file_wrapper;
+
+-- Replace the dummy schema with the actual schema of your file data
+CREATE FOREIGN TABLE local_file_table (x INT)
+SERVER local_file_server
+OPTIONS (path 'file:///path/to/file.parquet', extension 'parquet');
+```
+
+#### Local File Table Options
+
+- `path` (required): Must start with `file:///` and point to the location of your file. The path should end in a `/` if it points to a directory of partitioned Parquet files.
```

Comment:
Noticing that `file:///` is specified here, where above you use `s3://`. Is the triple-slash in `file` actually required for the url format, or is the third `/` just suggesting that the path be absolute? 

Does `file://` work with a relative file path? I think this would be helpful to clear up, especially as its non-obvious which directly the path would be relative to.

If only absolute paths are supported, that would be worth mentioning.